src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -491,8 +550,10 @@ describe('ThreadView', () => {
   });
 
   describe('for discussion thread', () => {
+    const findLoadMoreCommentsButton = () => screen.findByTestId('load-more-comments');
+
     it('shown post not found when post id does not belong to course', async () => {
-      renderComponent('unloaded-id');
+      await waitFor(() => renderComponent('unloaded-id'));
```

Comment:
I think we don't need waitFor here. recheck it please
await renderComponent('unloaded-id');

Reply:
without waitFor test case fails in PR